datahub-web-react/src/app/entity/dashboard/profile/DashboardProfile.tsx

```diff
@@ -67,6 +64,7 @@ export default function DashboardProfile({ urn }: { urn: string }) {
     return (
         <PageContainer>
             <>
+                {loading && <Message type="loading" content="Loading..." style={{ marginTop: '10%' }} />}
```

Comment:
do you want the margin to be a percentile? why not fixed?

Reply:
I think either works -- I just want the bubble to pop up somewhere near the top-middle of the page :)